RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHETimeSelected_cfi.py

```diff
@@ -9,112 +9,112 @@
             minEnergy = cms.double(0.0),
             maxEnergy = cms.double(2.0),
             endcap = cms.bool(False),
-            minTime = cms.double(-22.),
-            maxTime = cms.double(22.)
+            minTime = cms.double(-7.6),
```

Comment:
Seems like this should be -7.4 rather than -7.6 if I understand correctly the idea of this PR